### ISSUE TITLE
Updater never finishes updating this plugin.

### ISSUE THREAD

**Issue by SHoogland** [Glavin001/atom-beautify#761]
Hi, why is the update of this plugin always hanging or something. i have never seen the atom updater neatly finishing an atom-beatfiy update.

**Comment by tomck** [comment]
```
> spawn-sync@1.0.15 postinstall /private/var/folders/xf/kns5g7qd1nj8v1xs0qwf842h0000gp/T/apm-install-dir-116020-64040-14b42y5/node_modules/atom-beautify/node_modules/cross-spawn/node_modules/spawn-sync
> node postinstall


npm ERR! Darwin 15.2.0
npm ERR! argv "/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/Users/me/.atom/.apm/.apmrc" "--userconfig" "/Users/me/.atom/.apmrc" "install" "/private/var/folders/xf/kns5g7qd1nj8v1xs0qwf842h0000gp/T/d-116020-64040-afls6w/package.tgz" "--target=0.34.5" "--arch=x64"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! spawn-sync@1.0.15 postinstall: `node postinstall`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the spawn-sync@1.0.15 postinstall script 'node postinstall'.
npm ERR! This is most likely a problem with the spawn-sync package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node postinstall
npm ERR! You can get their info via:
npm ERR!     npm owner ls spawn-sync
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /private/var/folders/xf/kns5g7qd1nj8v1xs0qwf842h0000gp/T/apm-install-dir-116020-64040-14b42y5/npm-debug.log
```

I'd attach the log but it's not there. Also, my system versions are higher than it's using:

node -v
v5.4.1
npm -v
3.5.3

**Comment by prettydiff** [comment]
I wonder if simply updating Node and NPM would resolve this issue.  I don't see where node is install by the plugin though.  Is this an Atom dependency that's out of date?

**Comment by tomck** [comment]
I resolved the issue by simply updating in the terminal:

`apm install atom-beautify`

I assume this has access to the latest node/npm.

**Comment by Glavin001** [comment]
I just published a patch in release v0.29.4 that makes `cross-spawn` dependency optional. 
The [`cross-spawn`](https://github.com/IndigoUnited/node-cross-spawn) package depends on [`spawn-sync`](https://github.com/ForbesLindesay/spawn-sync) and has been causing problems when installing Atom-Beautify for many users. By making this dependency optional, users will be able to easily install Atom-Beautify and, in the case that [`cross-spawn`](https://github.com/IndigoUnited/node-cross-spawn) fails to install, will fallback to the builtin [`spawn`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) method.
